src/main/scala/sigmastate/serialization/BigIntConstantSerializer.scala

```diff
@@ -0,0 +1,24 @@
+package sigmastate.serialization
+
+import sigmastate.SBigInt
+import sigmastate.SType.TypeCode
+import sigmastate.Values.BigIntConstant
```

Comment:
By the way, not concerning this commit, does anybody know  why we use `java.math.BigInteger`, not `scala.math.BigInt` for `SBigIntConstant`?